mmdet/models/losses/focal_loss.py

```diff
@@ -179,3 +179,67 @@ def forward(self,
         else:
             raise NotImplementedError
         return loss_cls
+
+
+@LOSSES.register_module()
```

Comment:
Is there any difference with 
https://github.com/open-mmlab/mmdetection/blob/87eda061c466238cf2b746ec749777144499ba20/mmdet/models/losses/focal_loss.py#L106
If no, please use the version in mmdet

Reply:
I believe there are some small differences. This version introduced a relatively limited number if positive sample points, and only used these points to calculate positve loss.